packages/taquito-local-forging/src/error.ts

```diff
@@ -1,15 +1,6 @@
 import { ENTRYPOINT_MAX_LENGTH } from './constants';
 
-/**
- *  @category Error
- *  @description Error that indicates an invalid block hash being passed or used
- */
-export class InvalidBlockHashError extends Error {
-  public name = 'InvalidBlockHashError';
-  constructor(public message: string) {
-    super(message);
-  }
-}
+export { InvalidAddressError, InvalidHexStringError } from '@taquito/core';
```

Comment:
Didn't see InvalidAddressError export here previously, would like to understand why we'd like to export here now?

Reply:
thanks for the catch